Mathlib/Algebra/Group/Units/Defs.lean

```diff
@@ -262,6 +263,10 @@ lemma val_pow_eq_pow_val (n : ℕ) : ↑(a ^ n) = (a ^ n : α) := rfl
 @[to_additive (attr := simp, norm_cast)]
 lemma inv_pow_eq_pow_inv (n : ℕ) : ↑(a ^ n)⁻¹ = (a⁻¹ ^ n : α) := rfl
 
+@[to_additive (attr := simp)]
+lemma val_inv_inj : ((a⁻¹ : αˣ) : α) = ↑b⁻¹ ↔ (a : α) = b :=
```

Comment:
This is a lemma so it shouldn't be in a def file. This is probably why you got the large import.